README.md

```diff
@@ -52,10 +52,10 @@ or Wget:
 wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
 ```
 
-<sub>The script clones the nvm repository to `~/.nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc`, `~/.profile`, or `~/.bashrc`).</sub>
+<sub>The script clones the nvm repository to `$XDG_CONFIG_HOME/nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc`, `~/.profile`, or `~/.bashrc`).
 
 ```sh
-export NVM_DIR="$HOME/.nvm"
+export NVM_DIR="${XDG_CONFIG_HOME:-$HOME/config}/nvm"
```

Comment:
this seems like it'd put it in either `$XDG_CONFIG_HOME/nvm`, or `$HOME/config/nvm` - however, the locations i'd expect are `$XDG_CONFIG_HOME/.nvm` or `$HOME/.nvm`.